File: Basics_with_Python/PythonBasicCourse/2 Conditional Statements/6_Conditional_Statements_Excercises - Exercise/more_exercises/05. Pets.py
import math
days = int(input())
food_kg = int(input())
dog_daily_needs = float(input())
cat_daily_needs = float(input())
turt_daily_gr_needs = float(input())
turt_daily_needs = turt_daily_gr_needs / 1000
dog_daily_needs = dog_daily_needs * days
cat_daily_needs = cat_daily_needs * days
turt_daily_needs = turt_daily_needs * days
total_daily_needs = dog_daily_needs + cat_daily_needs + turt_daily_needs
difference = abs(total_daily_needs - food_kg)

if food_kg > total_daily_needs:
    print(f"{int(difference)} kilos of food left.")
else:
    print(f"{math.ceil(difference)} more kilos of food are needed.")

File: Basics_with_JavaScript/56_Conditional_Statements/More_Exersises/05._Pets.js
function vladi(input){

    let days = Number(input[0]);
    let availableFood = Number(input[1]);
    let dogPerDay = Number(input[2]);
    let catPerDay = Number(input[3]);
    let turtlePerDayGrams = Number(input[4]);
    let turtlePerDay = turtlePerDayGrams / 1000;

    let neededFood = Math.ceil((dogPerDay + catPerDay + turtlePerDay) * days)
    let difference = Math.ceil(Math.abs(availableFood - neededFood))
    
    if (availableFood >= neededFood) {
        console.log(`${difference} kilos of food left.`);
    } else {
        console.log(`${difference} more kilos of food are needed.`);
    }
    }

    vladi(["2", "10", "1", "1", "1200"]);
    vladi(["5", "10", "2.1", "0.8", "321"]);